lib/http/node.js

```diff
@@ -100,17 +106,19 @@ function isBypassProxy(proxy, destination) {
  * @inheritDoc
  */
 NodeHttpClient.prototype.getAgent = function (parsedUrl, proxy, mock) {
+  Logger.getInstance().trace('Agent[url: %s] - getting an agent instance.', parsedUrl.href);
```

Comment:
During additional testing in turned out that 'href' of parsedUrl can contain tokens as query params under path like '...queries/download-result?token=ver:1-hint:1036-...'.
Therefore the approached should be changed either to using only host and protocol info or to obfuscating the href on the fly to avoid exposing confidential data.

Reply:
Add token to mask secrets in logger.